.github/workflows/kind-upgrade.yml

```diff
@@ -23,8 +23,10 @@ jobs:
         name: antrea-ubuntu
         path: antrea-ubuntu.tar
 
-  from-v0_4_1:
-    name: Upgrade from Antrea v0.4.1
+# TODO: define an action to avoid repetition?
```

Comment:
Maybe we can have the actions testing upgrade from minor-2 and minor-1 releases (against the highest patch releases of these minor releases if we don't assume big change in patch releases?)

Reply:
I'll think of a way to do this programmatically for next release. Maybe it is actually enough to test with only the previous minor release.